### DISCUSSION TITLE
SSCursor.read_next() what is it?

### DISCUSSION THREAD

**Discussion by dashedman** [PyMySQL/PyMySQL#945] [Q&A] [answered]
What do SSCursor.read_next()? 
i didn't found this method in DB API (pep 249) =/

**Comment by methane** [comment, marked as answer]
It's internal function. Don't use it.

**Reply by dashedman** [reply]
Oh... Okay =)

You better use single leading undescore for naming this function

from PEP8, Naming Convention:
"In addition, the following special forms using leading or trailing underscores are recognized (these can generally be combined with any case convention):

    _single_leading_underscore: weak "internal use" indicator. E.g. from M import * does not import objects whose names start with an underscore.
"

**Comment by methane** [comment]
I know it, of course. I am not the original author of this library, but a just maintainer.
I added many leading underscores little by little.
